File: Rasha_Toine/Rasha/QLForm/src/qls/ast/widget/DropDown.java
package qls.ast.widget;

import java.util.List;

import qls.visiting.WidgetVisitor;

public class DropDown extends MultiOptionsWidget {
	//constructor
    public DropDown(List<String> options, String defaultOption) {
      super(options, defaultOption);
    }
    
	@Override
	public <T, U> T accept(WidgetVisitor<T, U> visitor, U ctx) {
		return visitor.visit(this, ctx);
	}
}

File: Erik_Vincent/Assignment1/Assignment1/Model/QLS/AST/Style/Widget/DropDown.cs
namespace Assignment1.Model.QLS.AST.Style.Widget
{
    public class DropDown : YesNoWidget
    {
        public DropDown(int lineNumber, string yes, string no) : base(lineNumber, yes, no) { }

        public override void Accept(IStyleVisitor visitor) => visitor.Visit(this);
    }
}
